### PR TITLE
Support multiple attachments

### LINKED ISSUE #28: Support multiple file attachments
attachment should be able to take an array like recipients or a singular

### PR DESCRIPTION
**Pull request by michaelschlies**
fixes #28

### CODE REVIEW

**Comment by mjwwit** [comment]
Thanks for the PR! ❤️
I noticed you replaced the `attachment` property in favor of `attachments`. This, however, will break all existing usages of this plugin that use the former syntax.
Would it be possible to support both?

**Comment by michaelschlies** [comment]
Updated

File: plugin.go [hunk 1]
```diff
@@ -6,7 +6,6 @@ import (
 	"github.com/aymerick/douceur/inliner"
 	"github.com/drone/drone-go/template"
 	"github.com/jaytaylor/html2text"
-	"gopkg.in/gomail.v2"
```
Comment (mjwwit) [inline plugin.go]: I just tested your code, and the removal of this import is giving me issues. Did you do this for a reason, or did it just slip through by accident?

Comment (michaelschlies) [reply plugin.go]: Ah, Goland's optimize imports got me somehow! I just readded this line.

**Comment by mjwwit** [comment]
Lovely, it now builds again and sending multiple attachments works like a charm! I'll merge and it'll be released shortly after.

**Pull request merged**